src/sphinx_argparse_cli/_logic.py

```diff
@@ -126,8 +128,12 @@ def run(self) -> list[Node]:
         return [home_section]
 
     def _mk_option_group(self, group: _ArgumentGroup, prefix: str) -> section:
-        title_text = f"{prefix}{' ' if prefix else ''}{group.title}"
-        ref_id = make_id(title_text)
+        group_title_prefix: str = (
+            prefix if self.options["group_title_prefix"] is None else self.options["group_title_prefix"]
```

Comment:
can you save self.options["group_title_prefix"] into a variable to fit this into one line. Also, do you need the annotation, I think would be automatically deduced by python from the right handside.